pkgs/by-name/ar/arc-browser/package.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, undmg
+, writeShellApplication
+, curl
+, common-updater-scripts
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "arc-browser";
+  version = "1.38.0-48670";
+
+  src = fetchurl {
+    url = "https://releases.arc.net/release/Arc-${finalAttrs.version}.dmg";
+    hash = "sha256-6LJEIkr1NA8HHxbLWtd/OTlPPErh05UTjVhjuZVcmTg=";
+  };
+
+  dontPatch = true;
+  dontConfigure = true;
+  dontBuild = true;
+  dontFixup = true;
```

Comment:
Probably the first three can be left enabled as they are noops.

Reply:
I'm little bit confused... Are you asking me to remove lines 19-21?

Reply:
Done

Reply:
Yes, if they don't interfere with the build, we should just keep the enabled.

Why don't we want to fixup here?

Reply:
Adressed